s2_jv_electric_boogaloo/healthBar.cpp

```diff
@@ -0,0 +1,60 @@
+#include "healthBar.h"
+
+healthBar::healthBar(int baseHealth)
+{
+	if (baseHealth > 0) {
+		this->health = baseHealth;
+	}
+}
+
+void healthBar::setHealth(int amount)
+{
+	if (amount > 0) {
+		this->health = amount;
+	}
+}
```

Comment:
MaxHealth n'est pas checked, on pourrait avoir 10 de vie mais 5 de max. 
```suggestion
void healthBar::setHealth(int amount)
{
	if (amount > 0) {
	        if (amount > this->getMaxHealth()) {
		        this->health = this->getMaxHealth();
	        } else {
		        this->health = amount;
	        }
	} else {
	        this->health = 0;
	}
}
```